### DISCUSSION TITLE
question about syntax

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [pmndrs/zustand#1114] [Q&A]
why does zustant use parentheses in the arrow function body
`increasePopulation: () => set((state) => ({ bears: state.bears + 1 }))`
can it be  `increasePopulation: () => set((state) => { bears: state.bears + 1 })` if it only returns one object ?

**Comment by dai-shi** [comment]
It's not the right JS syntax.

```
> const f = () => { a: 1 };
undefined
> f();
undefined
> const g = () => { a: 1, b: 2 };
const g = () => { a: 1, b: 2 };
                         ^

Uncaught SyntaxError: Unexpected token ':'
```
